spread.yaml

```diff
@@ -271,13 +271,13 @@ prepare: |
     fi
 
     # apt update is hanging on security.ubuntu.com with IPv6, prefer IPv4 over IPv6
-    cat <<EOF > /etc/gai.conf
-    precedence  ::1/128       50
-    precedence  ::/0          40
-    precedence  2002::/16     30
-    precedence ::/96          20
-    precedence ::ffff:0:0/96 100
-    EOF
+    #cat <<EOF > /etc/gai.conf
```

Comment:
Not at all, i added this changes for being able to test on core and forgot to remove it, now that the fix for core has landed there's a conflict in this PR related to these changes. I'll deconflict and repush, thanks!